From 63aa9d857bd54690020d6be2b20c48497c68e59f Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Fri, 6 Jul 2012 16:15:31 +0800
Subject: [PATCH 302/486] ENGR00215859 fix mx6sl Gallery preview picture
 flicker issue

fix mx6sl Gallery preview picture flicker issue
separate HAVE_FSL_IMX_GPU to HAVE_FSL_IMX_GPU3D and HAVE_FSL_IMX_GPU2D

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 mx5x/libcopybit/Android.mk        | 2 +-
 mx5x/libgralloc/Android.mk        | 4 +++-
 mx6/libgralloc_wrapper/Android.mk | 4 +++-
 3 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/mx5x/libcopybit/Android.mk b/mx5x/libcopybit/Android.mk
index d399ce0..f720147 100755
--- a/mx5x/libcopybit/Android.mk
+++ b/mx5x/libcopybit/Android.mk
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ifeq (true, false)
-ifeq ($(HAVE_FSL_IMX_GPU),true)
+ifeq ($(HAVE_FSL_IMX_GPU2D),true)
 ifeq ($(BOARD_SOC_CLASS),IMX5X)
 
 LOCAL_PATH := $(call my-dir)
diff --git a/mx5x/libgralloc/Android.mk b/mx5x/libgralloc/Android.mk
index e6812e7..071f5fa 100755
--- a/mx5x/libgralloc/Android.mk
+++ b/mx5x/libgralloc/Android.mk
@@ -45,7 +45,9 @@ endif
 
 ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
-else ifeq ($(HAVE_FSL_IMX_GPU),true)
+else ifeq ($(HAVE_FSL_IMX_GPU3D),true)
+LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
+else ifeq ($(HAVE_FSL_IMX_GPU2D),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
 endif
 
diff --git a/mx6/libgralloc_wrapper/Android.mk b/mx6/libgralloc_wrapper/Android.mk
index 385d63a..2efdd6f 100755
--- a/mx6/libgralloc_wrapper/Android.mk
+++ b/mx6/libgralloc_wrapper/Android.mk
@@ -40,7 +40,9 @@ LOCAL_C_INCLUDES = external/expat/lib
 
 ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
-else ifeq ($(HAVE_FSL_IMX_GPU),true)
+else ifeq ($(HAVE_FSL_IMX_GPU3D),true)
+LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
+else ifeq ($(HAVE_FSL_IMX_GPU2D),true)
 LOCAL_CFLAGS += -DFSL_IMX_DISPLAY
 endif
 
-- 
1.8.0

